web/test/pages/onboarding/onboarding-shared.test.tsx

```diff
@@ -43,6 +43,19 @@ describe("onboarding - shared", () => {
     expect(screen.getByTestId("step-1")).toBeInTheDocument();
   });
 
+  it.each(["business-persona-starting", "business-persona-foreign"])(
+    "allows %s to move past Step 1",
+    async (radioOption: string) => {
+      const { page } = renderPage({ userData: undefined });
+
+      page.chooseRadio(radioOption);
+      fireEvent.click(screen.getByTestId("next"));
+      await waitFor(() => {
+        expect(screen.getByTestId("step-2")).toBeInTheDocument();
+      });
+    }
+  );
+
   it("routes to the second onboarding page when they have answered the first question and we route them to page 2", async () => {
```

Comment:
[sand] I don't think this needs to be a part of this story (and if people think its not important I'm down to drop it) but it might be a good idea to have a test for the case JP was talking about to make sure that if we use the user data is reset and personas are updated accordingly if you switch between them on this screen. 

Example if I set myself as Oscar and choose a sector and then switch to poppy the sector I chose should probably be reset